tests/Feature/Api/GetProfilePostsTest.php

```diff
@@ -0,0 +1,98 @@
+<?php
+namespace Tests\Feature\Api;
+
+use Tests\TestCase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Facades\Artisan;
+use App\Models\User;
+use App\Models\UserInfo;
+
+class GetProfilePostsTest extends TestCase
+{
+    use DatabaseMigrations;
+
+    /**
+     * Set up product
+     *
+     * @return void
+     */
+    public function setUp()
+    {
+        parent::setUp();
+        factory('App\Models\Category')->create();
+        factory('App\Models\Product')->create();
+        factory('App\Models\User', 2)->create();
+        $inputId = User::doesntHave('userInfo')->pluck('id')->toArray();
+        $inputCount = count($inputId);
+        for ($i = 0; $i < $inputCount; $i++) {
+            factory('App\Models\UserInfo')->create([
+                'user_id' => $i+1,
+            ]);
+        }
+        factory('App\Models\Post', 20)->states('rating')->create();
+        Artisan::call('passport:install');
+    }
+
+    /**
+     * Return structure of json.
+     *
+     * @return array
+     */
+    public function jsonStructureGetPostsSuccess()
+    {
+        return [
+            'result' => [
+                'paginator' => [
+                    'current_page',
+                    'first_page_url',
+                    'from',
+                    'last_page',
+                    'last_page_url',
+                    'next_page_url',
+                    'path',
+                    'per_page',
+                    'prev_page_url',
+                    'to',
+                    'total'
+                ],
+                'data' => [
+                    [
+                        'id',
+                        'product_id',
+                        'user_id',
+                        'type',
+                        'content',
+                        'rating',
+                        'status',
+                        'created_at',
+                        'updated_at',
+                        'deleted_at',
+                    ]
+                ]
+            ],
+            'code'
+        ];
+    }
+
+    /**
+     * Test status code
+     *
+     * @return void
+     */
+    public function testGetPostsOfUser()
+    {
+        $user = User::find(1);
+        $login = [
+            'email' => $user->email,
+            'password' => '12345'
+        ];
+        $response = $this->json('POST', '/api/login', $login, ['Accept' => 'application/json']);
```

Comment:
Các test case ko nên phụ thuộc lẫn nhau. Khi em test cho `GET post` thì chỉ nên gọi api `GET post` để làm thôi.
Muốn lấy $token thì em tạo sẵn 1 $user, dùng $this->user để dùng lại nó. Nói lun là những gì chung, sử dụng lại nhiều lần cho các phần test khác nhau thì em đặt ở `TestCase` và dùng thôi, ko cần khai báo nhiều